diff --git a/chrome/browser/ui/views/frame/browser_view.cc b/chrome/browser/ui/views/frame/browser_view.cc
index d677978e498daa422c74fd2ecbb1f10190f34d20..d4728676f09c888aa9eda78b1994303b5e871642 100644
--- a/chrome/browser/ui/views/frame/browser_view.cc
+++ b/chrome/browser/ui/views/frame/browser_view.cc
@@ -261,7 +261,7 @@
 #include "ui/views/win/scoped_fullscreen_visibility.h"
 
 // To avoid conflicts with the macro from the Windows SDK...
-#undef LoadAccelerators
+//#undef LoadAccelerators
 #endif
 
 #if BUILDFLAG(ENABLE_ONE_CLICK_SIGNIN)
