diff --git a/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/SiteSettings.java b/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/SiteSettings.java
index 9783adbdc33b3dc398aa0533620b331530e3ff91..2a1b304eebf34f5f4fe19ded3266d060e3e7cb13 100644
--- a/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/SiteSettings.java
+++ b/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/SiteSettings.java
@@ -24,7 +24,7 @@ import org.chromium.content_public.browser.BrowserContextHandle;
  * browser-wide.
  */
 public class SiteSettings
-        extends SiteSettingsPreferenceFragment implements Preference.OnPreferenceClickListener {
+        extends BraveSiteSettingsPreferencesBase implements Preference.OnPreferenceClickListener {
     // The keys for each category shown on the Site Settings page
     // are defined in the SiteSettingsCategory.
 
